nixos/tests/web-servers/vencloud.nix

```
@@ -0,0 +1,28 @@
+import ../make-test-python.nix (
+  { lib, pkgs, ... }:
+  {
+    name = "vencloud";
+    meta.maintainers = with lib.maintainers; [ eveeifyeve ];
+
+    nodes.server = {
+      services.vencloud = {
+        enable = true;
+        settings = {
+          REDIS_URL = "redis://redis:vencloud@localhost:6379";
+        };
+      };
+
+      services.redis.servers."vencloud-redis" = {
+        enable = true;
+        openFirewall = true;
+      };
+    };
+
+    testScript = ''
+      server.start()
+      server.wait_for_unit("vencloud.service" "vencloud-redis.service")
```

```suggestion
      server.wait_for_unit("vencloud.service")
```
If the above is implemented, we shouldn't need to wait for both of these as the Redis service will be a dependency of the primary one